project/plugins.sbt

```diff
@@ -16,6 +16,6 @@ addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.23" exclude("org.slf4j",
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.5.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-service-manager" % "0.8.0")
```

Comment:
Ah, actually Bobby won't allow us to revert that one

Reply:
As https://build.tax.service.gov.uk/job/DDCW%20Live%20Services/job/Home%20Office%20APIs/job/individuals-employments-api-pr-builder/6/console confirmed, I think!